.vagrant/machines/default/virtualbox/action_provision

```diff
@@ -0,0 +1 @@
+1.5:3cbddbaa-fd55-4cd8-afa8-97502b4c4237
```

Comment:
You have quite a few vagrant files hanging around - probably from jumping between modules  - that we don't really want. We should remove these (either delete & commit that change, or just remove from git with `git rm --cached .vagrant -r`) and add `.vagrant` to your `.gitignore` (even if we will get it for free when we merge)